### PR TITLE
[INJI-697] Enhance Retrieve your ID screen

### PR DESCRIPTION
**Pull request by srikanth716**
Screenshot_1705061055
Screenshot_1705061063
Screenshot_1705061071

### CODE REVIEW

File: screens/Home/MyVcs/IdInputModal.tsx [hunk 1]
```diff
@@ -1,12 +1,18 @@
...
+import {
+  Dimensions,
+  I18nManager,
+  KeyboardAvoidingView,
+  TextInput,
+  View,
+} from 'react-native';
 import {TouchableOpacity} from 'react-native';
 import {individualId, isIOS} from '../../../shared/constants';
 import {GET_INDIVIDUAL_ID} from '../../../shared/constants';
```
Comment (KiruthikaJeyashankar) [inline screens/Home/MyVcs/IdInputModal.tsx:18]: Can we merge this imports to 
`import {individualId, isIOS, GET_INDIVIDUAL_ID} from '../../../shared/constants';`

File: screens/Home/MyVcs/IdInputModal.tsx [hunk 2]
```diff
@@ -80,6 +97,25 @@ export const IdInputModal: React.FC<IdInputModalProps> = props => {
                   selectionColor={Theme.Colors.Cursor}
                   value={controller.id}
                   keyboardType="number-pad"
+                  rightIcon={
+                    <Tooltip
```
Comment (KiruthikaJeyashankar) [inline screens/Home/MyVcs/IdInputModal.tsx:101]: Can we also extract this Tooltip to a component similar to tooltipContent?

Comment (KiruthikaJeyashankar) [reply screens/Home/MyVcs/IdInputModal.tsx:101]: Also if we having similar customisations for toolTip in other components as well (eg - GetIdInputModal) , can we create a ToolTip component in components/ui similar to Text component?
This way we can maintain consistency in our customization with Tooltip & avoid duplication

File: screens/Home/MyVcs/IdInputModal.tsx (hunk 1 above)
Comment (srikanth716) [reply screens/Home/MyVcs/IdInputModal.tsx]: merged the imports

File: screens/Home/MyVcs/IdInputModal.tsx (hunk 2 above)
Comment (srikanth716) [reply screens/Home/MyVcs/IdInputModal.tsx]: Created an custom tootip component in ui

**Review by KiruthikaJeyashankar — APPROVED** [review]

File: screens/Home/MyVcs/IdInputModal.tsx [hunk 3]
```diff
@@ -80,6 +89,21 @@ export const IdInputModal: React.FC<IdInputModalProps> = props => {
...
+                  rightIcon={
+                    <CustomTooltip
+                      title={t('toolTipTitle', {idType: controller.idType})}
+                      description={t(`toolTip${controller.idType}Description`)}
+                      width={Dimensions.get('screen').width * 0.85}
+                      height={Dimensions.get('screen').height * 0.18}
+                      triggerComponent={
+                        <Icon
+                          name="infocirlceo"
+                          type="antdesign"
```
Comment (KiruthikaJeyashankar) [inline screens/Home/MyVcs/IdInputModal.tsx:101]: Apologies for being, can we also add testIDs for the tooltip for automation testing?

Comment (swatigoel) [reply screens/Home/MyVcs/IdInputModal.tsx:101]: @srikanth716 let's add testID to this tooltip view.
testID should be passed as prop. This is ui component will be used at multiple places, so testID should be contextual.

**Review by swatigoel — APPROVED** [review]

**Review by swatigoel — APPROVED** [review]

**Pull request merged**